packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -400,12 +403,29 @@ class TapDragEndDetails with Diagnosticable {
   /// the number in the series this tap is.
   final int consecutiveTapCount;
 
+  /// The global position at which the pointer lifted from the screen.
+  /// Similar to [LongPressEndDetails].
```

Comment:
You can remove the line mentioning `LongPressEndDetails`, sorry if that was unclear.